### PR TITLE
Add Git icons to the GithubEvent card title

### PR DESCRIPTION
**Pull request by dgparmar14**
I add icons to the "GithubEvent card title" as per the git hub actions mentioned

### CODE REVIEW

File: app/contributors/dgparmar14.md
Comment (rithviknishad) [inline app/contributors/dgparmar14.md:1]: Discard this file. You can add your markdown to https://github.com/coronasafe/leaderboard-data repo. This repo does not and should not contain any specific data as other organizations would be using this as source code to deploy their data.

File: contributors
Comment (rithviknishad) [inline contributors:1]: This symlink is required for leaderboard to correctly point to the data source

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: app/contributors/dgparmar14.md
Comment (rithviknishad) [reply app/contributors/dgparmar14.md:1]: Tomorrow morning your markdown profile will be auto-generated by the scrapper workflow. You could also simply edit the existing markdown too then.

File: contributors
Comment (rithviknishad) [reply contributors:1]: Still unresolved

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by dgparmar14** [comment]
Okay i will do it

**Pull request closed without merging**